src/tools/quemap/monitor.c

```diff
@@ -259,4 +277,5 @@ void Mon_Shutdown(const char *msg) {
 	mon_backlog = NULL;
 
 	Net_Shutdown();
+#endif
```

Comment:
Was libxml2 too difficult to wrangle into VS? I gotta be honest, without the XML monitoring, quemap is pretty useless on Windows. The XML monitoring is how it talks to GtkRadiant. It's absolutely stupid, but this is now GtkRadiant does it.